lisa/sut_orchestrator/libvirt/qemu_platform.py

```diff
@@ -39,6 +40,28 @@ def _create_node_os_disk(
         self, environment: Environment, log: Logger, node: Node
     ) -> None:
         node_context = get_node_context(node)
+        # if environment.runbook.nodes_requirement:
+        #     node_req: List[NodeSpace] = environment.runbook.nodes_requirement
+        #     for _, node_space in enumerate(node_req):
+        #         node_runbook: BaseLibvirtNodeSchema = node_space.get_extended_runbook(
+        #             self.node_runbook_type(),
+        #             type_name=type(self).type_name(),
+        #         )
+
+        #         if node_runbook.disk_img_resize_gib:
+        #             qemu_tool = self.host_node.tools[QemuImg]
+        #             src_file = node_context.os_disk_base_file_path
+        #             qemu_tool.resize(
+        #                 src_file=src_file,
+        #                 size_gib=node_runbook.disk_img_resize_gib,
+        #             )
+        node_runbook: BaseLibvirtNodeSchema = node.runbook
+        if node_runbook.disk_img_resize_gib:
+            qemu_tool = self.host_node.tools[QemuImg]
+            qemu_tool.resize(
```

Comment:
This should be moved to be after the `create_diff_qcow2` call (as per my other comment).

Reply:
I would also do the same thing in `CloudHypervisorPlatform`.

Reply:
Done